runtime/src/lib.rs

```diff
@@ -148,6 +148,7 @@ parameter_types! {
 	/// We allow for 5 MB blocks.
 	pub BlockLength: frame_system::limits::BlockLength = frame_system::limits::BlockLength
 		::max_with_normal_ratio(5 * 1024 * 1024, NORMAL_DISPATCH_RATIO);
+	pub const SS58Prefix: u8 = 42;
```

Comment:
I'm fine with this, but I find it weird that we need to add it, considering we are not supporting SS58

Reply:
Should this match the chainId btw ? if yes, we have a problem !

Reply:
Yeah, it won't be visibile at all on moonbeam because we use ethereum addresses. But it's something frame_system expects, so we have to give it. I agree it isn't the most general design.